test/REQUIRE

```diff
@@ -0,0 +1 @@
+Dates
```

Comment:
That shouldn't be necessary, `Dates` is a stdlib, and those don't need to be listed in `REQUIRE`.